fixtures-generator/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.8-slim
+
+COPY requirements.txt .
```

Comment:
I don't think it is necessary to get rid of that file or to make that intermediate build stage we talked about last week just to get rid of that file. This `Dockerfile` is very easy to understand like this and works perfectly. The other way would complicate it a lot and running the fixtures generation would be more complicated (I experienced issues on missing python modules; maybe the virtual environment needs to be activated in the other solution). But anyway, I think we can keep it simple and stupid here, it's just fixtures generation, nothing going to production.